setuptools_odoo/make_default_setup.py

```diff
@@ -318,9 +319,8 @@ def clean_setup_addons_dir(addons_dir, odoo_version_override):
         paths_to_remove.append(metapackage_dir)
 
     # XXX we may want to clean metapackage_dir/setup.cfg if not universal_wheel
-
-    paths_to_remove = [os.path.abspath(p) for p in paths_to_remove]
-    subprocess.check_call(["rm", "-rf"] + paths_to_remove)
+    for p in paths_to_remove:
+        shutil.rmtree(os.path.abspath(p))
```

Comment:
Thanks for the PR. 

Note that not all paths_to_remove are directories, so this will likely not work.

Can you also add a file named `110.bugfix` with a one-sentence changelog entry.

Reply:
;)

I hope that now it's ok.